### PR TITLE
Makman2/docfix

### PR DESCRIPTION
**Pull request by Makman2**

### CODE REVIEW

File: coalib/bearlib/abstractions/Linter.py [hunk 1]
```diff
@@ -612,9 +612,9 @@ def linter(executable: str,
         A custom command to check for when ``check_prerequisites`` gets
         invoked (via ``subprocess.check_call()``). Must be an ``Iterable``.
     :param prerequisite_check_fail_message:
-        A custom command to check for when ``check_prerequisites`` gets
-        invoked. Must be provided only together with
-        ``prerequisite_check_command``.
+        A custom message that gets displayed when ``check_prerequisites``
+        fails when invoking ``prerequisite_check_command``. Can only be
```
Comment (Adrianzatreanu) [inline coalib/bearlib/abstractions/Linter.py]: could you use while instead of when? it hurts my psychically having when x when y; too many ``when``s

**Comment by Adrianzatreanu** [comment]
ack f4192aacf

File: coalib/bearlib/abstractions/Linter.py [hunk 2]
```diff
@@ -608,6 +613,9 @@ def linter(executable: str,
         created when ``generate_config`` is supplied. Useful if your executable
         expects getting a specific file-type with specific file-ending for the
         configuration file.
+    :param executable_check_fail_info:
+        Information that is provided together with the fail message from the
+        normal executable check. By default no additional info is printed.
```
Comment (Adrianzatreanu) [inline coalib/bearlib/abstractions/Linter.py]: i suppose this one can be provided without the ``prerequisite_check_command``, right?

Comment (Makman2) [inline coalib/bearlib/abstractions/Linter.py]: yes

**Comment by Makman2** [comment]
reack 522720e

**Comment by Adrianzatreanu** [comment]
ack e808812

**Comment by Adrianzatreanu** [comment]
ack 522720e

**Comment by Adrianzatreanu** [comment]
ack 3e9e115

**Comment by Makman2** [comment]
@rultor merge

**Pull request merged**